### PR TITLE
Add text box

### PR DESCRIPTION
**Pull request by eireland**

### CODE REVIEW

**Comment by eireland** [comment]
The really relevant file is the secondary-embeddable.tsx, scss, the activity-page-content.tsx, and the text-box.tsx, scss. The other embeddables are along for the ride and will need to be worked on more.

File: cypress/support/elements/activity-page.js [hunk 1]
```diff
@@ -11,5 +14,10 @@ class ActivityPage {
   getSidebarCloseButton() {
     return cy.get("[data-cy=sidebar-close-button");
   }
+  getSecondaryEmbeddable(type) { 
+    //types=["image-question","text-box","multiple-choice-question","open-response-question", 
+    //      "labbook-question", "iframe-interactive-question","image-video-interactive", ]
```
Comment (mklewandowski) [inline cypress/support/elements/activity-page.js:19]: is this obsolete?

File: src/components/app.scss [hunk 2]
```diff
@@ -21,5 +21,6 @@
   height: fit-content;
   padding: 0 20px;
   background-image: linear-gradient(to bottom, #fff, rgba(255,255,255,0), rgba(255,255,255,0));
+  line-height: 1.35;
+  color: #444;
```
Comment (mklewandowski) [inline src/components/app.scss:25]: should this be a predefined color in vars?

File: src/components/app.tsx [hunk 3]
```diff
@@ -6,7 +6,9 @@ import { ActivityPageContent } from "./activity-page/activity-page-content";
 import { IntroductionPageContent } from "./activity-introduction/introduction-page-content";
 import Footer from "./activity-introduction/footer";
 import { PageLayouts } from "../utilities/activity-utils";
-import sampleActivity from "../data/sample-activity-multiple-layout-types.json";
+// import sampleActivity from "../data/sample-activity-multiple-layout-types.json";
+import sampleActivity from "../data/LARA-page-elements-full-width-off.json";
+// import sampleActivity from "../data/LARA-page-elements-full-width-on.json";
```
Comment (mklewandowski) [inline src/components/app.tsx:11]: Probably can remove these commented lines.

File: cypress/support/elements/activity-page.js (hunk 1 above)
Comment (eireland) [reply cypress/support/elements/activity-page.js:19]: No, it's the way Kevin and I keep track of what the parameters should look like.

Comment (mklewandowski) [reply cypress/support/elements/activity-page.js:19]: Ah, got it!

File: src/components/app.scss (hunk 2 above)
Comment (eireland) [reply src/components/app.scss:25]: Added both to the vars file. #444 as medium-font-color. line height as line-height.

File: src/components/app.tsx (hunk 3 above)
Comment (mklewandowski) [reply src/components/app.tsx:11]: Sounds good!

**Pull request merged**